helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -149,6 +149,18 @@ rules:
       - "/metrics"
     verbs:
       - get
+  - apiGroups:
+      - autoscaling.k8s.io
+    resources:
+      - verticalpodautoscalers
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
why do we need `watch`?

Reply:
Well spotted, I just copied from something else and selected too much. Will fix.

Reply:
Done.